lib/pdksync.rb

```diff
@@ -6,11 +6,14 @@
 require 'pdk'
 require 'pdksync/constants'
 require 'pdksync/gitplatformclient'
+require 'pdksync/jenkinclient'
 require 'json'
 require 'yaml'
 require 'colorize'
 require 'bundler'
 require 'octokit'
+require 'HTTParty'
```

Comment:
octokit uses faraday already.  I would drop httparty and use faraday instead.

Reply:
Hey @logicminds , thank you. i'll update it.